content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst

```diff
@@ -2,40 +2,123 @@
+
+Once the frequency has been changed to the desired number of days, click :guilabel:`Save` to save
+changes. Now, once an inventory adjustment is applied to this location, the next scheduled count
+date is automatically set, based on the value entered into the :guilabel:`Inventory Frequency
+(Days)` field.
+
+Count inventory by location
+===========================
+
+To perform a cycle count for a specific location in the warehouse, navigate to
+:menuselection:`Inventory app --> Operations --> Inventory Adjustments`. This reveals an
+:guilabel:`Inventory Adjustments` page containing all products currently in stock, with each product
+listed on its own line.
+
+From this page, the :guilabel:`Filters` and :guilabel:`Group By` buttons (at the top of the page,
+under the :guilabel:`Search...` bar), can be used to select specific locations and perform inventory
+counts.
+
+.. image:: cycle_counts/cycle-counts-inventory-adjustments-page.png
    :align: center
-   :alt: Enable Storage Locations in Odoo settings.
+   :alt: Inventory adjustments page.
 
-Change the inventory frequency
-==============================
+To select a specific location and view all products within that location, click :guilabel:`Group
+By`, then click :guilabel:`Add Custom Group` to reveal a new drop-down menu to the right.
 
-To change a location's inventory frequency, first, go to the locations by clicking
-:menuselection:`Inventory --> Configuration --> Locations`.
+Click :guilabel:`Location` from the drop-down menu, then click :guilabel:`Apply`. The page now
+displays condensed drop-down menus of each location in the warheouse that has products in stock, and
+a cycle count can be performed for all products in that location.
 
-Then, click on a location to open the location settings. Next, click on :guilabel:`Edit`. In the
-:guilabel:`Inventory Frequency (Days)` field, set the number of days. For example, a location that
-needs an inventory count every 30 days would set the :guilabel:`Inventory Frequency (Days)` value
-to `30`. Once the value is entered, click :guilabel:`Save` to apply the setting to the location.
-Now, once an inventory adjustment is applied to this location, the next scheduled count date will
-be automatically set based on the number of days in the :guilabel:`Inventory Frequency (Days)`
-setting.
+.. tip::
+   In large warehouses with a large number of locations and high volume of products, it might be
+   easier to search for the specific location desired. To do this, from the
+   :menuselection:`Inventory Adjustments` page, click :guilabel:`Filters`. Then, click
+   :guilabel:`Add Custom Filter` to reveal a new menu to the right. Click this menu to reveal three
+   drop-downs.
 
-.. image:: cycle_counts/inventory-frequency.png
+   For the first field, click and select :guilabel:`Location` from the drop-down. For the second
+   field, leave the `contains` value as is. For the third field, type in the name of the location
```

Comment:
I think this value should be :guilabel: since it is not user input
```suggestion
   field, leave the :guilabel:`contains` value as is. For the third field, type in the name of the
```